app/views/announcements/_announcement.html.slim

```diff
@@ -11,11 +11,6 @@
 
     = format_inline_text(announcement.title)
 
-    div.pull-right
```

Comment:
Why's this removed?

Reply:
The edit and delete path no longer exist for system announcements, so it didn't make sense to render them.